### PR TITLE
NEWS: The Advent of Void: Day 16: darkhttpd

### PR DESCRIPTION
**Pull request by jnbr**

### CODE REVIEW

File: _posts/2018-12-16-advent-darkhttpd.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+to more machines for testing, before pushing them to the `void-packages`:
+
+```
+$ darkhttpd hostdir/binpkgs/firefox
+darkhttpd/1.12, copyright (c) 2003-2016 Emil Mikulic.
+listening on: http://0.0.0.0:8080/
+```
+
+`darkhttpd` by defaults listens on port 8080, so to update the testing machines,
+running `sudo xbps-install --repository=http://buildbox:8080 -Suy` is all to be done.
```
Comment (Vaelatern) [inline _posts/2018-12-16-advent-darkhttpd.markdown]: Have you tried this? I suspect not :P

Comment (Vaelatern) [reply _posts/2018-12-16-advent-darkhttpd.markdown]: Most of us don't automatically sign our packages, which is why I express doubt.

Comment (jnbr) [reply _posts/2018-12-16-advent-darkhttpd.markdown]: I used it with signed packages. See log below :)

Comment (jnbr) [reply _posts/2018-12-16-advent-darkhttpd.markdown]: added a note that packages must be signed for this to work.

**Pull request merged**